crates/types/src/config/bifrost.rs

```diff
@@ -89,7 +89,7 @@ impl BifrostOptions {
 impl Default for BifrostOptions {
     fn default() -> Self {
         Self {
-            default_provider: ProviderKind::Local,
+            default_provider: ProviderKind::Replicated,
```

Comment:
I think this change might be a bit problematic because if users start with `1.2` and then roll back to `1.1` because of some problems, then they by default be using the replicated loglet which in `1.1` wasn't really implemented if I recall correctly.